docs/device_report.md

```diff
@@ -127,7 +128,9 @@ RESULT fail connection.mac_oui
 ## Module bacext
 
 ```
-RESULT skip protocol.bacnet.version
+RESULT skip protocol.bacnet.version Bacnet device not found.
+RESULT skip protocol.bacnet.pic
+Bacnet device not found... Pics check cannot be performed.
```

Comment:
@JoelRedstone is there another rogue carriage return here?

Reply:
yep

Reply:
this same line description situation is not applicable to existing tests. project management advice

Reply:
It seems to already be done for this test, but agreed